common/utils/hwcutils.cpp

```diff
@@ -22,15 +22,17 @@
 
 namespace hwcomposer {
 
-void HWCPoll(int fd, int timeout) {
+int HWCPoll(int fd, int timeout) {
```

Comment:
@munish-b Can you update commit message as per this:
https://github.com/intel/IA-Hardware-Composer/wiki/Contributions

Reply:
I have updated the commit message.